composer.json

```diff
@@ -21,15 +21,16 @@
         "source": "https://github.com/xiidea/EasyAuditBundle"
     },
     "require": {
-        "php": "^7.2.5",
+        "php": "^8.0.0",
```

Comment:
`"php": "^8.0.0",` will prevent from installing in php7

Reply:
updated with ^7.2.5